intake/catalog/base.py

```diff
@@ -9,9 +9,11 @@
 import logging
 import re
 import time
+from tkinter import W
```

Comment:
I didn't notice this before.
This line isn't needed right? tkinter isn't amongst Intake's requirements.

Reply:
Not sure how that got in there. Probably an accidental auto-import. I'll remove it.

Reply:
I think I spotted one further down too

Reply:
(oh, you got it)